eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/JournalStatements.scala

```diff
@@ -175,7 +184,7 @@ object JournalStatements {
               val payload = payloadType.map {
                 case PayloadType.Binary => row.decode[Option[Payload.Binary]]("payload_bin") getOrElse Payload.Binary.empty
                 case PayloadType.Text   => row.decode[Payload.Text]("payload_txt")
-                case PayloadType.Json   => row.decode[Payload.Json]("payload_txt")
+                case PayloadType.Json   => row.decode[Payload.Json]("payload_txt")(decodeByNameJson)
```

Comment:
why explicit ?

Reply:
Instance of `DecodeByName[Payload.Json]` is cached to avoid creation for each call. The explicit usage is for making this detail more explicit ;)